docs/instruction_for_testing.md

```diff
@@ -0,0 +1,135 @@
+# How to run full integration test suite
+
+The purpose of this document is to detail the steps to run the project's integration tests on each supported platform. Not all test flows have been integrated in our automated pipeline so it is important to follow this guide and understand how to run tests in a semi-automatically fashion.
+
+It is **important** to note that this guide is not strictly for running tests while prepping for a release. This guide should be reviewed and the testing process defined here should be run before a release, as you incrementally add logic. That way, if bugs are uncovered, you have enough time to address them!
+
+## Deployment and Setup
+
+Before we can run our integration tests, we need to deploy the required environments and create the test artifact.
+
+At a high level, you will need to:
+
+1. Deploy and configure machines
+1. Install dependencies
```

Comment:
Create development environment and install requirements

Reply:
Done